src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -86,7 +86,7 @@ void sl_ble_init()
                                    NULL, NULL, NULL);
 
     // registering the GATT call back functions
-    rsi_ble_gatt_register_callbacks(NULL, NULL, NULL, NULL, NULL, NULL, NULL, rsi_ble_on_gatt_write_event, NULL, NULL, NULL,
+    rsi_ble_gatt_register_callbacks(NULL, NULL, NULL, NULL, NULL, NULL, NULL, rsi_ble_on_gatt_write_event, NULL, NULL, rsi_ble_on_read_req_event,
```

Comment:
For the sake of efficiency, couldn't we reduce the number of parameter at the same time ?

Reply:
yes this seems unbearable !